pkgs/applications/video/shotcut/default.nix

```diff
@@ -45,14 +48,12 @@ mkDerivation rec {
     qtwebsockets
     qtquickcontrols2
     qtgraphicaleffects
+    fftw
   ];
 
   env.NIX_CFLAGS_COMPILE = "-I${mlt.dev}/include/mlt++ -I${mlt.dev}/include/mlt";
-  qmakeFlags = [
-    "QMAKE_LRELEASE=${lib.getDev qttools}/bin/lrelease"
-    "SHOTCUT_VERSION=${version}"
-    "DEFINES+=SHOTCUT_NOUPGRADE"
-  ];
+
+  SHOTCUT_VERSION = "${version}";
```

Comment:
We have to put this into cmakeFlags:
`cmakeFlags = [ "-DSHOTCUT_VERSION=${version}" ];`